oe_webtools_analytics/oe_webtools_analytics.behat.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * @file
+ * Contains \OEParagraphsSubContext.
+ */
+
+declare(strict_types = 1);
+
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\DrupalExtension\Context\DrupalSubContextBase;
+use Drupal\oe_webtools_analytics\Utils\WebtoolsAnalyticsIndex;
+
+/**
+ * Behat step definitions for testing the OE Webtools Analytics module.
+ */
+class OEWebtoolsAnalyticsSubContext extends DrupalSubContextBase {
+
+  /**
+   * Setup demo site.
```

Comment:
Better: install module (site is already installed)

Reply:
We will have the following issue when using Drupal Extension Behat context autoloading (i.e. when contexts are defined in a `.behat.inc`): if `OEWebtoolsAnalyticsSubContext` uses a trait or a class that is autoloaded via the `autoload-dev` property of the module's `composer.json` such trait or class will not be available when the `oe_webtools` module will be used as a dependency in another project; at the same time the Drupal Extension will include the class anyway, which will result in a fatal.

Since we use a Composer-based workflow we don't need third-party autoloading methods, we can just use Composer `autoload` and `autoload-dev` properties.